tests/rt_utils.sh

```diff
@@ -270,9 +270,25 @@ check_results() {
         d=$( cmp ${RTPWD}/${CNTL_DIR}/$i ${RUNDIR}/$i | wc -l )
 
         if [[ $d -ne 0 ]] ; then
-          echo ".......NOT OK" >> ${REGRESSIONTEST_LOG}
-          echo ".......NOT OK"
-          test_status='FAIL'
+
+          if [[ ${MACHINE_ID} =~ orion || ${MACHINE_ID} =~ hera ]]; then
+            printf ".......NOT OK...TRY ALT CHECK.." >> ${REGRESSIONTEST_LOG}
+            printf ".......NOT OK...TRY ALT CHECK"
```

Comment:
We should not have a "NOT OK" and then and "OK" in the same log file.
There will be a "NOT OK" or an "OK"